www/js/main.js

```diff
@@ -2773,6 +2774,20 @@ function updateWeather() {
 	} );
 }
 
+function checkURLandUpdateWeather() {
+	return $.get( currPrefix + currIp + "/su" ).then( function( reply ) {
```

Comment:
Only thought is that it would be better for Firmware to publish `wsp` and `jsp` as standard controller options. That way we wouldn't need to parse the html and we could also add it to "Import/Export Config" so that it can survive an upgrade.

If we thought that was a good idea then we could future proof by wrapping this code with a `if (!controller.options.wsp) {...}`  and add the options to firmware "/js" code.

Reply:
Okay support for 2.1.9 has been added to this PR and I changed the default to http.